components/email_aliases/email_aliases_auth.cc

```diff
@@ -0,0 +1,127 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/email_aliases/email_aliases_auth.h"
+
+#include "base/auto_reset.h"
+#include "base/base64.h"
+#include "components/os_crypt/sync/os_crypt.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+
+namespace {
+
+bool Encrypt(const std::string& plain_text, std::string& out) {
+  if (plain_text.empty()) {
+    return false;
+  }
+
+  if (!OSCrypt::EncryptString(plain_text, &out)) {
```

Comment:
this should be replaced with async OSCrypt. New code should not use synchronous interface.

`components/os_crypt/sync/README.md`
> This is the legacy 'sync' interface which should not be used in new code. Prefer using the 'async'
interface for any new code.